src/server/pkg/obj/obj.go

```diff
@@ -360,12 +375,13 @@ func (b *BackoffWriteCloser) Write(data []byte) (int, error) {
 			return err
 		}
 		return nil
-	}, b.backoffConfig, func(err error, d time.Duration) {
+	}, b.backoffConfig, func(err error, d time.Duration) error {
```

Comment:
Weird... how was this compiling before?

Reply:
It was using a different library. One of the changes in this PR is that I replaced `github.com/cenkalti/backoff` with `github.com/pachyderm/pachyderm/src/server/pkg/backoff`

Reply:
Ahh, right. Makes sense.